### ISSUE TITLE
Value of g

### ISSUE THREAD

**Issue by ketch** [ketch/circular_hydraulic_jump#10]
We state the value of g for one of the experiments (g=1 in 5.1) but not for all of them.  If we use g=1 everywhere, we can just write this at the beginning of section 5.

**Comment by manuel-quezada** [comment]
I am trying to handle this issue with the new PR. Once that PR is merged we can close this issue.
